requests_mock/__init__.pyi

```diff
@@ -0,0 +1,7 @@
+# Stubs for requests_mock
+
+from requests_mock.adapter import ANY as ANY, Adapter as Adapter
+from requests_mock.exceptions import MockException as MockException, NoMockAddress as NoMockAddress
+from requests_mock.mocker import DELETE as DELETE, GET as GET, HEAD as HEAD, Mocker as Mocker, MockerCore as MockerCore, OPTIONS as OPTIONS, PATCH as PATCH, POST as POST, PUT as PUT, mock as mock
+from requests_mock.request import _RequestObjectProxy as _RequestObjectProxy
+from requests_mock.response import CookieJar as CookieJar, create_response as create_response
```

Comment:
Why all the `as` items for things that have the same name?

Reply:
See the note on stubs in https://mypy.readthedocs.io/en/stable/command_line.html#cmdoption-mypy-no-implicit-reexport